x-pack/legacy/plugins/alerting/server/alerts_client.test.ts

```diff
@@ -845,6 +799,121 @@ describe('create()', () => {
       }
     );
   });
+
+  test(`doesn't create API key for disabled alerts`, async () => {
+    const data = getMockData({ enabled: false });
+    alertsClientParams.createAPIKey.mockResolvedValueOnce({
```

Comment:
I don't think this is needed, given 

```ts
expect(alertsClientParams.createAPIKey).not.toHaveBeenCalled();
```

But I also suspect it could be useful if we ever start calling this, so ... it's fine.  Perhaps a comment that it's not expected this will actually get called ...

Reply:
Nice catch! I went ahead and removed it here: https://github.com/elastic/kibana/pull/57041/commits/31ead6d9fa2fb14d76749cbef05d0ebfec68b799 and maybe in a future refactor this code will move into a `beforeEach` for the ones that do call that function.